.gitignore

```diff
@@ -1,4 +1,9 @@
 .idea/
 **/.DS_Store
+/java/Anton-Kozyk/Anton-Kozyk.iml
```

Comment:
1. Please remove `/java/Anton-Kozyk/Anton-Kozyk.iml` from the file
2. `/out/` is defined twice